apps/services/auth/admin-api/src/app/v2/clients/me-clients.controller.ts

```diff
@@ -74,7 +75,7 @@ export class MeClientsController {
     @CurrentUser() user: User,
     @Param('tenantId') tenantId: string,
     @Param('clientId') clientId: string,
-    @Param('includeArchived') includeArchived?: boolean,
+    @Query('includeArchived') includeArchived?: boolean,
```

Comment:
Was this intentional ?

Reply:
yes, it should be a query param and to a param, the open api yaml file had errors, so it would not work with Postman, me and Sævar made these changes to fix them.